sources/accessible-name-and-description.ts

```diff
@@ -538,6 +538,18 @@ export function computeTextAlternative(
 			return "Submit Query";
 		}
 
+		if (hasAnyConcreteRoles(node, ["button"])) {
```

Comment:
`<a href>` elements have a similar requirement.  I don't get the impression that that is addressed yet in this PR.  See https://www.w3.org/TR/html-aam-1.0/#a-element-accessible-name-computation

Reply:
Can do in a follow-up